http/src/request/channel/reaction/create_reaction.rs

```diff
@@ -49,17 +49,53 @@ impl<'a> CreateReaction<'a> {
         }
     }
 
+    fn request(&self) -> Request {
+        Request::from(Route::CreateReaction {
+            channel_id: self.channel_id.0,
+            emoji: self.emoji.clone(),
+            message_id: self.message_id.0,
+        })
+    }
```

Comment:
please don't add a convenience function here, instead leave that part of the file as is and manually create the needed model in test, or add a convenience function to the test module if desired

Reply:
disregard this